README.md

```diff
@@ -4,11 +4,15 @@ Spectrum Kit is a collection of TypeScript tools designed for various scenarios.
 
 ## Table of Contents
 
-- [Introduction](#introduction)
-- [Installation](#installation)
-- [Usage](#usage)
-- [Contributing](#contributing)
-- [License](#license)
+- [Spectrum Kit](#spectrum-kit)
+  - [Table of Contents](#table-of-contents)
+  - [Introduction](#introduction)
+    - [BrowserHistory](#browserhistory)
+    - [DoubleLinkedListNode](#doublelinkedlistnode)
+    - [Installation](#installation)
+    - [Usage](#usage)
+    - [Contributing](#contributing)
+    - [License](#license)
```

Comment:
I think these needs to be the same level as Introduction, so be ## instead of ### for Usage, Contributing and License. Then these three can be the same amount of tab spacing as introduction.